drivers/hmis/spec/requests/hmis/ce/ce_referral_spec.rb

```diff
@@ -541,13 +541,16 @@
           before do
             remove_permissions(ds_access_control, :can_perform_any_referral_tasks)
             add_permissions(ds_access_control, :can_perform_own_referral_tasks)
+            referral.participants.create(swimlane: case_manager_swimlane, user: hmis_user)
+
+            referral.workflow_engine.start_workflow!(user: hmis_user)
           end
 
           it 'still does not cause n+1' do
             expect do
               _, result = post_graphql(**variables) { query }
               steps = result.dig('data', 'ceReferral', 'steps')
-              expect(steps.map { |step| step.dig('access', 'canPerformStep') }).to all(be false)
+              expect(steps.map { |step| step.dig('access', 'canPerformStep') }).to all(be true)
```

Comment:
So this is expected to be `true` now because the `hmis_user` is now assigned to the swimlane for the open steps, is that right?